### PR TITLE
Allows viewing delete/copy iso/template buttons for non ready resources and enables settings

### PR DESCRIPTION
**Pull request by Pearl1594**
Addresses : https://github.com/apache/cloudstack-primate/issues/511

### CODE REVIEW

**Comment by Pearl1594** [comment]
@blueorangutan package

File: src/components/view/DetailSettings.vue [hunk 1]
```diff
@@ -173,7 +173,8 @@ export default {
     },
     isAdminOrOwner () {
       return ['Admin'].includes(this.$store.getters.userInfo.roletype) ||
-        (this.resource.domainid === this.$store.getters.userInfo.domainid && this.resource.account === this.$store.getters.userInfo.account)
+        (this.resource.domainid === this.$store.getters.userInfo.domainid && this.resource.account === this.$store.getters.userInfo.account) ||
+        this.resource.project && this.resource.project === this.$store.getters.project.name
```
Comment (rhtyd) [inline src/components/view/DetailSettings.vue:177]: @Pearl1594 should the check be on projectid instead?

**Comment by Pearl1594** [comment]
My bad, @rhtyd addressed it

**Comment by Pearl1594** [comment]
@blueorangutan package

**Pull request merged**